apps/src/maze/skins.js

```diff
@@ -47,6 +47,9 @@ var CONFIGS = {
     movePegmanAnimationSpeedScale: 1.5,
     // This is required when move pegman animation is set
     movePegmanAnimationFrameNumber: 9,
+    actionSpeedScale: {
+      'nectar': 2,
```

Comment:
No need to have nectar in quotes (we're unfortunately inconsistent about this in our code in general, but I'd like to have new code be quoteless unless necessary).